i18n/en.json

```diff
@@ -2805,8 +2805,8 @@
   "signup_user_completed.required": "This field is required",
   "signup_user_completed.reserved": "This username is reserved, please choose a new one.",
   "signup_user_completed.signIn": "Click here to sign in.",
-  "signup_user_completed.userHelp": "Username must begin with a letter, and contain between {min} to {max} lowercase characters made up of numbers, letters, and the symbols '.', '-' and '_'",
-  "signup_user_completed.usernameLength": "Username must begin with a letter, and contain between {min} to {max} lowercase characters made up of numbers, letters, and the symbols '.', '-' and '_'.",
+  "signup_user_completed.userHelp": "You can use lowercase letters, numbers, periods, dashes, and underscores",
```

Comment:
Should this message end with a trailing `.`?

Reply:
@esethna What do you think about this?

Reply:
@esethna: ^ ping

Reply:
Yes it should contain a period. Sorry missed the notifications here, thanks for the ping